Taskwarrior Pomodoro/AppDelegate.swift

```diff
@@ -367,11 +472,12 @@ class AppDelegate: NSObject, NSApplicationDelegate, NSMenuDelegate {
     func startTaskById(taskId: String) {
         activeTaskId = taskId
         
-        let task = NSTask()
-        task.launchPath = taskPath
-        task.arguments = [taskId, "start"]
-        task.launch()
-        task.waitUntilExit()
+        taskCommand([taskId, "start"])
+        activeTaskPomodorosLogUUID = currentPomodorosLogUUID
```

Comment:
For those of us that leave our computers on overnight, I think this UUID would be inaccurate the next day, no?